fixtures/stairville/wild-wash-648-led-rgb-dmx.json

```
@@ -0,0 +1,383 @@
+{
+  "$schema": "https://raw.githubusercontent.com/OpenLightingProject/open-fixture-library/master/schemas/fixture.json",
+  "name": "Wild Wash 648 LED RGB DMX",
+  "shortName": "WildWash648LEDRGB",
+  "categories": ["Color Changer", "Strobe", "Blinder"],
+  "meta": {
+    "authors": ["Paul Rogalski"],
+    "createDate": "2024-10-13",
+    "lastModifyDate": "2024-10-13"
+  },
+  "links": {
+    "manual": [
+      "https://images.static-thomann.de/pics/atg/atgdata/document/manual/c_399664_399663_399658_399662_v3_en_online.pdf"
+    ],
+    "productPage": [
+      "https://www.thomann.de/intl/stairville_wild_wash_648_led_rgb_dmx.htm"
+    ],
+    "video": [
+      "https://www.thomann.de/intl/thomanntv_video_stairville_wild_wash_6100.html"
+    ]
```

```suggestion
    ]
```

Dead link.

---

@kengruven Did you copied it correctly? It works pretty well for me...

---

Hmm, maybe it's country dependent. I'm gonna change it to a different one

---

We can link to the video directly:

```suggestion
    ],
    "video": [
      "https://video1.thomann.de//vidiot/02591c1c/video_i6100p8_yd59vq0a.mp4"
    ]
```

---

That one works for me.